scalability_and_performance/scaling-cluster-monitoring-operator.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view system resources, containers, and components metrics in one dashboard interface, Grafana.
+{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the Prometheus-based monitoring stack. As an administrator, you can view dashboards for system resources, containers, and components metrics in the {product-title} web console by navigating to *Observe* -> *Dashboards*.
```

Comment:
Maybe use the same term for the stack here:
```suggestion
{product-title} exposes metrics that the Cluster Monitoring Operator collects and stores in the the Prometheus Cluster Monitoring stack. As an administrator, you can view dashboards for system resources, containers, and components metrics in the {product-title} web console by navigating to *Observe* -> *Dashboards*.
```

Reply:
I'm leaving this as-is. We don't capitalize the word "monitoring" in the docs, and we don't use the phrase "cluster monitoring stack" anywhere else. Generally, we just refer to the "monitoring stack" and saying `Prometheus-based monitoring stack` isn't inconsistent with this practice.

Reply:
That makes sense. I had made that recommendation based on the opening paragraph, which included "Prometheus Cluster Monitoring," and the following instances elsewhere in the openshift-docs repo:

Results for "Prometheus Cluster Monitoring" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/whats_new/new-features.adoc:
  82:  |Prometheus cluster monitoring


/home/rdlugyhe/openshift-docs/modules/cluster-logging-deploy-cli.adoc:
  48:  with metrics, you should configure the Prometheus Cluster Monitoring stack to scrape metrics from the `openshift-operators-redhat`


/home/rdlugyhe/openshift-docs/modules/infrastructure-moving-monitoring.adoc:
  10:  By default, the Prometheus Cluster Monitoring stack, which contains Prometheus, Grafana, and AlertManager,


/home/rdlugyhe/openshift-docs/modules/machine-config-daemon-metrics.adoc:
  8:  provides a set of metrics. These metrics can be accessed using the Prometheus Cluster Monitoring stack.